### ISSUE TITLE
nginx or php-fpm7.3 problem after install OMV in Raspberry pi 2B

### ISSUE THREAD

**Issue by nodaro** [OpenMediaVault-Plugin-Developers/docs#24]
I follow the full manual, and when I try to acces to OMV via web by IP, it returns "502 Bad Gateway nginx".
I can see in the installation log, the next about php services (i write the last failed):
```
 ID: test_phpfpm_service_config
    Function: cmd.run
        Name: php-fpm7.3 --test
      Result: False
     Comment: Command "php-fpm7.3 --test" run
     Started: 07:39:52.621077
    Duration: 355.073 ms
     Changes:
              ----------
              pid:
                  9962
              retcode:
                  -4
              stderr:
              stdout:
----------
          ID: restart_phpfpm_service
    Function: service.running
        Name: php7.3-fpm
      Result: False
     Comment: One or more requisite failed: omv.deploy.phpfpm.default.test_phpfpm_service_config
     Started: 07:39:53.279794
    Duration: 0.132 ms
     Changes:
----------
          ID: monitor_phpfpm_service
    Function: module.run
        Name: monit.monitor
      Result: False
     Comment: One or more requisite failed: omv.deploy.phpfpm.default.restart_phpfpm_service
     Started: 07:39:53.308567
    Duration: 0.132 ms
     Changes:

Summary for raspberrypi
------------
Succeeded: 2 (changed=2)
Failed:    3
------------
Total states run:     5
Total run time:  28.455 s
```

Any suggestion?

**Comment by nodaro** [comment]
I try to install in a raspberry pi 3, and it works.
Is this manual able to pir 2B?

**Comment by flmaxey** [comment]
I'm testing builds right now.  R-PI 2B builds are so slow, we're thinking about dropping support.  It seems apparent that R-PI2B's are too under powered to used as a NAS.

**Comment by flmaxey** [comment]
OK, I've built an R-PI 2B according the guide with no issues.  The Dev of the script has done a few builds as well.  In essence, we can't duplicate your result.  

With that said, I can offer a theory as to what might have happened because I did it myself:

Are you sure you're building an R-PI 2B?  I pulled a model "B+" off of the shelf (not supported), by accident, and got the same result - 502 Bad Gateway nginx.  The R-PI 2B and the B+ are physically (nearly) identical.  The only apparent difference, at first glance, is the R-PI 2B has a chip on the bottom side of the board where the B+ does not.  

Check the version just above the Raspberry logo printed on the board.
